src/com/owncloud/android/datamodel/FileDataStorageManager.java

```diff
@@ -1847,7 +1847,47 @@ public void saveSharesInFolder(ArrayList<OCShare> shares, OCFile folder) {
                 c = null;
             }
         }
-        ArrayList<OCShare> shares = new ArrayList<OCShare>();
+        ArrayList<OCShare> shares = new ArrayList<>();
```

Comment:
I would rename this variable to privateShares in order to clearly know what are we getting from the database